pkg/cmd/cli/restore/get.go

```diff
@@ -59,6 +59,13 @@ func NewGetCommand(f client.Factory, use string) *cobra.Command {
 				cmd.CheckError(err)
 			}
 
+			// Change phase to "Deleting" if deletionTimestamp is marked.
+			for i := range restores.Items {
+				if !restores.Items[i].DeletionTimestamp.IsZero() {
+					restores.Items[i].Status.Phase = "Deleting"
+				}
```

Comment:
Similar with this section. Instead of here, lets modify restore_printer.go:
https://github.com/vmware-tanzu/velero/blob/main/pkg/cmd/util/output/restore_printer.go#L60
```
I don't think this will work, since "Deleting" isn't a valid value for this field. A better place to do this would be here:
https://github.com/vmware-tanzu/velero/blob/main/pkg/cmd/util/output/restore_describer.go#L46

Also, I don't think we want to lose the actual value of the field (i.e. completed vs. failed), so maybe in restore_describe.go, something like this:

	if !restore.DeletionTimestamp.IsZero() {
		status = "Deleting"
	}
```
Here we can just use Deleting rather than showing prior status since it's just a quick summary.

Reply:
@allenxu404 I guess the only real concern was that I wasn't sure that would work since you're setting Phase to something that's not defined as a `RestorePhase` const, but maybe that's fine since we're not actually *updating* the restore object in the cluster. If the compiler is happy with this and it's working for you there, then it should be fine.